package.json

```diff
@@ -21,7 +32,11 @@
     "url": "git+https://github.com/cisagov/crossfeed.git"
   },
   "scripts": {
-    "start": "docker compose build --parallel && docker compose up --force-recreate"
+    "lint": "eslint '**/*.{ts,tsx,js,jsx}'",
```

Comment:
Shouldn't need to change this file except for your es addition. These are all the packages installed in the frontend lambda so we should be careful to add anything that isn't necessary. 

This all might be fine just think we should triple-check any changes here.